.github/workflows/pr-snapshot.yaml

```diff
@@ -45,7 +45,12 @@ jobs:
               issue_number: process.env.PR_NUMBER,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: `Snapshot successfully published to "oci://${process.env.ECR_ACCOUNT_ID}.ecr.${process.env.ECR_REGION}.amazonaws.com/karpenter/snapshot/karpenter:v0-${process.env.PR_COMMIT}".`
+              body: `Snapshot successfully published to "oci://${process.env.ECR_ACCOUNT_ID}.dkr.ecr.${process.env.ECR_REGION}.amazonaws.com/karpenter/snapshot/karpenter:v0-${process.env.PR_COMMIT}".
```

Comment:
Looks like this may need to be changed now that the environment variables have udpated.